addons/account/models/account.py

```diff
@@ -177,6 +193,28 @@ def action_open_reconcile(self):
         }
 
 
+class AccountGroup(models.Model):
+    _name = "account.group"
+
+    _parent_store = True
+
+    parent_id = fields.Many2one('account.group', index=True, ondelete='cascade')
+    parent_left = fields.Integer('Left Parent', index=True)
+    parent_right = fields.Integer('Right Parent', index=True)
+    name = fields.Char(required=True)
+    code_prefix = fields.Char()
+
+    @api.depends('name', 'code_prefix')
```

Comment:
@jorenvo 
Could it be that in case one of these fields change that the cache is invalidated for this method, which I guess is only needed in case the method would be used as a source for a computed field, right? So, the "what" is certainly reasonable :-)

But I just learned that the depends can be valuable even for not stored, computed fields (usage of cache vs. always recomputing on call). No idea, but maybe there is another hidden feature? :-) @rco-odoo